components/tools/OmeroPy/src/omero/__init__.py

```diff
@@ -104,6 +104,14 @@ def proxy_to_instance(proxy_string, default=None):
     """
     import omero
     parts = proxy_string.split(":")
+    # If a default is provided and the format is Class:ID
```

Comment:
> Maybe there is an argument for adding an additional enforce_default flag to the method?

That sounds more non-breaking. Alternative naming suggestion might be to use:

```
 def proxy_to_instance(proxy_string, default=None, enforce=None):
```
And either:
 * only one of default and enforce should be used *or*
 * the enforce check is just applied on top of everything else.